bundles.md

```diff
@@ -0,0 +1,131 @@
+# Attestation Bundle
+in-toto attestations support multiple different predicate types with the idea that
```

Comment:
I find this intro a bit confusing since you mention predicates, then assertions, then attestations. Maybe just start with "An attestation bundle is a collection of multiple [attestations](...) in a single file." and then explain why it's useful?

Reply:
Done WDYT?